RadixWallet/Features/DerivePublicKeys/DerivePublicKeys.swift

```diff
@@ -181,6 +211,22 @@ public struct DerivePublicKeys: Sendable, FeatureReducer {
 			)
 		}
 	}
+
+	public func reduce(into state: inout State, childAction: ChildAction) -> Effect<Action> {
+		switch childAction {
+		case .factorSourceAccess(.delegate(.perform)):
+			.run { send in
+				/// For more information about that `sleep` please  check [this discussion in Slack](https://rdxworks.slack.com/archives/C03QFAWBRNX/p1687967412207119?thread_ts=1687964494.772899&cid=C03QFAWBRNX)
+				@Dependency(\.continuousClock) var clock
+				try? await clock.sleep(for: .milliseconds(700))
+				await send(.internal(.delayedStart))
```

Comment:
I am not sure if we need to keep the delay now that the process is moved from a view pushed into a navigation into a sheet modally presented.